python/cudf/cudf/core/column/lists.py

```diff
@@ -285,3 +285,36 @@ def take(self, lists_indices):
             raise
         else:
             return res
+
+    def unique(self):
+        """
+        Returns unique element for each list in the column, order for each
+        unique element is not guaranteed.
+
+        Returns
+        -------
+        ListColumn
+
+        Examples
+        --------
+        >>> s = cudf.Series([[1, 1, 2, None, None], None, [4, 4], []])
+        >>> s
+        0    [1.0, 1.0, 2.0, nan, nan]
```

Comment:
Is the `None` --> `nan` just a side effect of how `repr` is currently implemented?

`pandas.Series.unique` also handles deduplicating `nan` and `<NA>` values. We should at least note the difference in behavior here, but we should probably raise an issue for libcudf to have an option for `null` / `nan` handling.

Reply:
Yup, `__repr__` depends on pyarrow and:
```python
>>> pa.array([[1, 2, None, 4]]).to_pandas()
0    [1.0, 2.0, nan, 4.0]
dtype: object
```
I can verify under the hood cudf hasn't change the dtype.